<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Contest</class>
 <widget class="QWidget" name="Contest">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
    <widget class="Label_time" name="lab_countdown">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Countdown</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
    <widget class="QLabel" name="lab_ctname">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Contest_name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_ctpwd">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>45</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="verticalSpacing">
        <number>20</number>
       </property>
       <item row="2" column="1">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" colspan="2" alignment="Qt::AlignmentFlag::AlignHCenter">
          <widget class="QLineEdit" name="linee_ctpwd">
           <property name="minimumSize">
            <size>
             <width>248</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;
border: 1px solid rgba(0, 0, 0, 0.3);
border-radius: 10px;
color:grey;
</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1" alignment="Qt::AlignmentFlag::AlignLeft">
          <widget class="QLabel" name="lab_pwdwrong">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>密码错误</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QPushButton" name="pu_ctpwd">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton { 
    font: 12pt 'Microsoft YaHei UI'; 
    color: white; 
    border-radius: 15px; 
    background: linear-gradient(to bottom right, rgba(255,255,255,0.2), transparent 50%), 
               qlineargradient(x1: 1, y1: 0, x2: 1, y2: 1, stop: 0 #ffa060, stop: 1 #c08040); 
}

QPushButton:hover { 
    background: linear-gradient(to bottom right, rgba(255,255,255,0.3), transparent 50%), 
               qlineargradient(x1: 1, y1: 0, x2: 1, y2: 1, stop: 0 #ffb070, stop: 1 #d09050); 
}

QPushButton:pressed { 
    background: linear-gradient(to bottom right, rgba(0,0,0,0.1), transparent 50%), 
               qlineargradient(x1: 1, y1: 0, x2: 1, y2: 1, stop: 0 #c08040, stop: 1 #a06020); 
    padding: -2px -2px 2px 2px; 
}</string>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>257</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>47</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>257</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>请输入比赛密码</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_ctsign">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="1">
        <widget class="QPushButton" name="pu_sign">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>160</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: white;
    border-radius: 76px;
    background: linear-gradient(to bottom right, rgba(255,255,255,0.2), transparent 50%), 
               qlineargradient(x1:1, y1:0, x2:1, y2:1, stop:0 #50c050, stop:1 #30a030);
}

QPushButton:hover {
    background: linear-gradient(to bottom right, rgba(255,255,255,0.3), transparent 50%), 
               qlineargradient(x1:1, y1:0, x2:1, y2:1, stop:0 #60d060, stop:1 #40b040);
}

QPushButton:pressed {
    background: linear-gradient(to bottom right, rgba(0,0,0,0.1), transparent 50%), 
               qlineargradient(x1:1, y1:0, x2:1, y2:1, stop:0 #308030, stop:1 #206020);
}</string>
         </property>
         <property name="text">
          <string>签到</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>164</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>333</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>333</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_ctmain">
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>80</y>
         <width>331</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string>比赛详细</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Label_time</class>
   <extends>QLabel</extends>
   <header location="global">rewritten_widgets/label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
